doc/src/manual/performance-tips.md

```diff
@@ -601,7 +601,7 @@ This should be written as:
 
 ```jldoctest
 julia> function fill_twos!(a)
-           for i=1:length(a)
+           for i=linearindices(a)
```

Comment:
Would it make more sense to use `eachindex(a)`?

Reply:
agreed that `eachindex` is more intuitive. But the offset-arrays doc recommend `linearindices` (https://docs.julialang.org/en/latest/devdocs/offset-arrays) so I chose that for consistency.
Should I
1. keep `linearindices` both places for consistency
2. change `linearindices`->`eachindex` both places for consistency (for this use - iterating linearly over a container)
3. change `linearindices`->`eachindex` here and not worry about the small discrepancy
?
I have no clear preference.

Reply:
Use `linearindices` if you really want to index with an integer. If you don't care what you index with, use `eachindex`.

Reply:
I see what you mean about it being ambiguous in the devdocs.

Reply:
I've updated to `eachindex` here. I don't mind including some changes to the devdocs here, or we sould call it good as it is?

Reply:
If you're willing I'd be grateful, but feel free to say "your job, Tim." :smile:

Reply:
I've tried here: https://github.com/JuliaLang/julia/pull/23731 . Can't say I'm completely happy with the result at this point, but possibly after some feedback that could work.